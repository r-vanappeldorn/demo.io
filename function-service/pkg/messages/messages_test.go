package messages

import (
	"testing"

	"github.com/gkampitakis/go-snaps/snaps"
)

func TestError(t *testing.T) {
	snaps.MatchSnapshot(t, Error())
}

func TestSuccess(t *testing.T) {
	snaps.MatchSnapshot(t, Succes())
}