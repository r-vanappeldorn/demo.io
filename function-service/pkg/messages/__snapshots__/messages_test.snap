
[TestError - 1]

[error]
---

[TestSuccess - 1]
[success]
---
